editors/code/src/Commands/compileSM.ts

```diff
@@ -1,18 +1,18 @@
 import {
-  workspace as Workspace,
+  workspace,
```

Comment:
Please keep it with a capital or import the whole `vscode` module. The reason is that `workspace` can be a variable in the file and this can cause name collisions.

Reply:
Understood!